app/pages/Accounts/AccountReport/AccountReport.tsx

```diff
@@ -149,6 +163,13 @@ export default function AccountReport({ location }: Props) {
                 content={showError.content}
                 onClick={() => setShowError({ show: false })}
             />
+            <SimpleErrorModal
```

Comment:
It's quite hacky to use one of our error modals for an info message. We should use another component for this.

Reply:
I've used the `MessageModal` instead